### PR TITLE
Finish ex1

### PR DESCRIPTION
**Pull request by Exg020**

### CODE REVIEW

File: src/main/java/com/xpanxion/java/assignments/Main.java [hunk 1]
```diff
@@ -3,6 +3,6 @@
 public class Main {
 
     public static void main(String[] args) {
-       Switch.run();
+       com.xpanxion.java.assignments.Switch.run();
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/assignments/Main.java:6]: Does this need to be here?

File: src/main/java/com/xpanxion/java/assignments/student9/Worker9.java [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
 public class Worker9 {
 
+
+
     public void ex1() {
-        System.out.println("Ex. 1...");
+
+        Map<Integer, Department> departmentMap = DataAccess.getDepartments().stream()
+                .collect(Collectors.toMap(Department::getId, Function.identity()));
+        ;
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/assignments/student9/Worker9.java:20]: Look at this lonely semicolon. :-)

File: src/main/java/com/xpanxion/java/assignments/Main.java (hunk 1 above)
Comment (Exg020) [reply src/main/java/com/xpanxion/java/assignments/Main.java:6]: it does not i fixed it

File: src/main/java/com/xpanxion/java/assignments/student9/Worker9.java (hunk 2 above)
Comment (Exg020) [reply src/main/java/com/xpanxion/java/assignments/student9/Worker9.java:20]: Fixed it

File: src/main/java/com/xpanxion/java/assignments/Main.java (hunk 1 above)
Comment (mburolla) [reply src/main/java/com/xpanxion/java/assignments/Main.java:6]: Hum. I still see three files in this PR?  There should only be two, right?

**Pull request closed without merging**